__DEFINES/weapons.dm

```diff
@@ -26,3 +26,17 @@
 #define LAWGIVER_MODE_KIND_ENERGY "energy"
 #define LAWGIVER_MODE_KIND_BULLET "bullet"
 #define LAWGIVER_MAX_AMMO 5
+
+//gun shit - prepare to have various things added to this, also, moved here because it's more tidy
+#define SILENCECOMP  1 		//Silencer-compatible
+#define AUTOMAGDROP  2		//Does the mag drop when it's empty?
+#define EMPTYCASINGS 4		//Does the gun eject empty casings?
+#define SCOPED		 8		//Attachable scope?
+#define CHAMBER 	 16		//Revolvers usually don't release casing when shot, only when reloaded
```

Comment:
Could you make this define name more descriptive? All guns have chambers after all